lib/gtfsplus/containers/ActiveGtfsPlusEditor.js

```diff
@@ -47,17 +60,20 @@ const mapDispatchToProps = (dispatch, ownProps) => {
       if (!initialProps.feedSource) dispatch(fetchFeedSourceAndProject(feedSourceId))
       if (!initialProps.tableData) dispatch(downloadGtfsPlusFeed(feedVersionId))
     },
-    newRowClicked: (tableId) => dispatch(addGtfsPlusRow(tableId)),
+    newRowClicked: (payload) => dispatch(addGtfsPlusRow(payload)),
```

Comment:
For all of the single-argument actions, you don't need to add a dispatch.  You can simply say `newRowClicked: addGtfsPlusRow,`

Reply:
I don't think this is true.  This only works if you're specifying a POJO for `mapDispatchToProps`.

Reply:
oh, bummer

Reply:
I know... it's pretty awful.  Maybe I should just fix the chained dispatches.